packages/firestore/src/api/timestamp.ts

```diff
@@ -64,10 +108,24 @@ export class Timestamp {
     }
   }
 
+  /**
+   * Convert a Timestamp to a JavaScript `Date` object. This conversion causes
+   * a loss of precision since `Date` objects only support millisecond precision.
+   *
+   * @return JavaScript `Date` object representing the same point in time as
+   *     this `Timestamp`, with millisecond precision.
+   */
   toDate(): Date {
     return new Date(this.toMillis());
   }
 
+  /**
+   * Convert a timestamp to a numeric timestamp (in milliseconds since epoch).
```

Comment:
"Converts"

Also, the first "timestamp" should probably be `Timestamp` since we're talking about both this type and "numeric timestamp" in the same sentence.

Reply:
Fixed